deployment/k8sPaiLibrary/maintainlib/deploy.py

```diff
@@ -106,6 +107,26 @@ def job_executer(self, node_config, job_name):
 
 
 
+    def update_node_config(self):
+        cluster_config = self.cluster_config
+        node_config_from_cluster_conf = dict()
+
+        for role in cluster_config["remote_deployment"]:
```

Comment:
En, will need the configuration layout and explanation in order to understand.

Reply:
No, It's by design. For different k8s role.